src/common/create-error/plural-module-presence.js

```diff
@@ -0,0 +1,9 @@
+define([
+	"../create-error"
+], function( createError ) {
+
+return function() {
+	return createError( "E_MISSING_PLURAL_MODULE", "Plural module not loaded." );
```

Comment:
As an organizational point of view for source files, all custom errors are placed there.

As a performance point of view for distribution files, this function becomes an inlined variable. So, there's no extra function wrapping.